src/styles-reset.ts

```diff
@@ -0,0 +1,82 @@
+import { css } from "styled-components"
+
+export const stylesReset = css`
+  /* Box sizing rules */
+  *,
+  *::before,
+  *::after {
+    box-sizing: border-box;
+  }
+
+  /* Remove default padding */
+  ul[class],
+  ol[class] {
+    padding: 0;
+  }
+
+  /* Remove default margin */
+  body,
+  h1,
+  h2,
+  h3,
+  h4,
+  p,
+  ul[class],
+  ol[class],
+  li,
+  figure,
+  figcaption,
+  blockquote,
+  dl,
+  dd {
+    margin: 0;
+  }
+
+  /* Set core body defaults */
+  body {
+    min-height: 100vh;
+    scroll-behavior: smooth;
+    text-rendering: optimizeSpeed;
+    line-height: 1.5;
+  }
```

Comment:
i agree with having the styles-reset, but shouldn't `body` styles be placed in the specific app?

Reply:
They should be placed in the specific app, it's just a module to import and use, if you like. For instance, here in storybook app, I use them through GlobalStyles styled-components API. Other people may or may not use it, it's not "built-in" in the ui-kit, just like media queries etc.

Reply:
I understand, i had a feeling that `min-height: 100vh` is too specific for the individual page, and it can be problematic to treat it as an "universal solution" for us. The common style reset is very helpful and it would be not good to create separate resets in our apps, but on the other hand, if needed, we can override problematic style if it will be needed.

Reply:
We can even remove the problematic style, as this dependency is meant to be used by us on the first place =)